1_fetch.yml

```diff
@@ -4,18 +4,21 @@ include:
   - lib.yml
 
 packages:
+  - dataRetrieval
   - httr
```

Comment:
i think we'll need `yaml` in here. i originally hoped that packages listed in lib.yml would be available to all inheriting yml files, but i think i learned that's not the case.